src/components/Quote.js

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from 'react';
+
+function Quote() {
+  const [quote, setQuote] = useState('');
+  const [author, setAuthor] = useState('');
+  const [category, setCategory] = useState('');
+  const [error, setError] = useState('error');
+  const [loading, setLoading] = useState('not loading');
+  const fetchQuote = async () => {
+    const response = await fetch('https://api.api-ninjas.com/v1/quotes', {
+      headers: {
+        'X-Api-Key': 'nezpEokeOVTwP1C5NwtGEQ==h62LjJXfJNexoQfy',
+      },
+    });
+    const data = await response.json();
+    const { quote, author, category } = data[0];
+    setQuote(quote);
+    setAuthor(author);
+    setCategory(category);
+    setError('not error');
+    setLoading('loading');
+  };
+
+  useEffect(() => {
+    fetchQuote();
+  }, []);
```

Comment:
- [ ] There is a need for the useEffect to rerun at points when some data changes and it should be setLoading and setQuote so the array should not be empty, kindly fill them appropriately.